spark/v3.2/spark/src/test/java/org/apache/iceberg/spark/actions/TestRewriteDataFilesAction.java

```diff
@@ -24,6 +24,7 @@
 import java.io.UncheckedIOException;
 import java.util.Collections;
 import java.util.Comparator;
+import java.util.HashMap;
```

Comment:
We use `Maps.newHashMap` when we are creating maps so usually we don't need to import the class directly

Reply:
Got that.